client/asset/dcr/spv.go

```diff
@@ -690,7 +695,34 @@ func (w *spvWallet) GetTransaction(ctx context.Context, txHash *chainhash.Hash)
 	if txd.Block.Height != -1 {
 		ret.BlockHash = txd.Block.Hash.String()
 		ret.Confirmations = int64(tipHeight - txd.Block.Height + 1)
+	} else {
+		// Check with TxConfirms and TxBlock now, why not.
+		confs, err := w.dcrWallet.TxConfirms(ctx, txHash)
+		if err != nil {
+			w.log.Warnf("TxConfirms: %v", err)
+		} else if confs > 0 {
+			w.log.Warnf("TxConfirms says yes! %d confs", confs)
+		}
+		blockHash, blockHeight, err := w.dcrWallet.TxBlock(ctx, txHash)
+		if err != nil {
+			w.log.Warnf("TxBlock: %v", err)
+		} else if blockHeight > 0 {
+			w.log.Warnf("TxBlock says yes! block %v, height %d", blockHash, blockHeight)
+			ret.BlockHash = blockHash.String()
+			ret.Confirmations = int64(tipHeight - blockHeight + 1)
+			goto gotit
+		}
+		_, confs2, blockHashP, err := w.dcrWallet.TransactionSummary(ctx, txHash)
+		if err != nil {
+			w.log.Warnf("TransactionSummary: %v", err)
+		} else if confs2 > 0 {
+			w.log.Warnf("TransactionSummary says yes! block %v, confs %d", blockHashP, confs2)
+			ret.BlockHash = blockHashP.String()
+			ret.Confirmations = int64(confs2)
+			goto gotit
```

Comment:
All of this is because I wanted to see if any of the other `wallet` methods can give a conf or block inspite of `TxDetails`.  It does not appear to return any different results.  This seems to be because they all stop on `existsRawUnmined`, just like `TxDetails`.